index.html

```diff
@@ -485,57 +485,64 @@ <h5 class="pa-h1-v3 pa-bright pt-4 pb-1 mb-4 fadeInDelay03Duration10">Let <stron
             <!-- Section Header -->
             <div class="row text-center">
                 <div class="col-12 col-xl-10 offset-xl-1">
-                    <h1 class="pa-h1-v2 pa-bright fadeInDelay00Duration10">Let's <strong>Connect</strong></h1>
-                    <p class="pa-p-v1 pa-white fadeInDelay03Duration10">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero. Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet. Duis sagittis ipsum. Praesent mauris.</p>
+                    <h1 class="pa-h1-v2 pa-bright fadeInDelay00Duration10"><strong>Contact Us</strong></h1>
```

Comment:
please make it 
![image](https://user-images.githubusercontent.com/29180114/126026947-cbe007e3-2eb3-40c5-b84f-e810f963c041.png)

Reply:
done